tests/e2e/support/api/keycloak/utils.ts

```diff
@@ -30,7 +30,8 @@ export const refreshToken = async ({ user }: { user: User }): Promise<void> => {
   const tokenEnvironment = TokenEnvironmentFactory('keycloak')
 
   const body = new URLSearchParams()
-  body.append('client_id', 'security-admin-console')
+  // client-id `admin-cli` has Direct access grants in keycloak setup and used for password grant type
```

Comment:
hard to understand 😄 plz rephrase

Reply:
I have no other way of rephrasing that. It's just to explain why such client id is used. If anyone want, they can go to keycloak setup and see the scope of client id on their own.

Reply:
# Using Keycloak's `admin-cli` client with Direct Access Grants for password grant type
# Allows exchanging username/password for an access token directly via the token endpoint

Reply:
This was comment in the line where request is made.

Reply:
` // only client-id `admin-cli` has Direct access grants, thus for password grant type, this client_id is recommended.` was your previous comment